include/albatross/src/utils/async_utils.hpp

```diff
@@ -17,6 +17,8 @@ namespace albatross {
 
 namespace detail {
 
+static constexpr std::size_t cDefaultStackSize{8 * 1024 * 1024};
```

Comment:
defaulting to using GNU Linux's default of 8MiB. In one of the runs GM runs I recorded 2.1MiB of stack usage.

Reply:
Is it possible to leave this out of albatross and just expose the ability to set it, then set it appropriately from other programs that use albatross?

Reply:
I'll update the interface to see if we can pass the configuration up the stack

Reply:
all done